common/client/versionChecker.go

```diff
@@ -56,6 +56,8 @@ const (
 	GoWorkerConsistentQueryVersion = "1.5.0"
 	// JavaWorkerRawHistoryQueryVersion indicates the minimum client version of the java worker which supports RawHistoryQuery
 	JavaWorkerRawHistoryQueryVersion = "1.3.0"
+	// JavaWorkerConsistentQueryVersion indicates the minimum client version of the java worker which supports RawHistoryQuery
```

Comment:
nit: please update the comment `RawHistoryQuery` -> `consistent query`

Reply:
Yeah fix it